tests/test_schema.py

```diff
@@ -746,6 +746,28 @@ def test_schema_duplicate_def_03(self):
             };
         """
 
+    @tb.must_fail(
+        errors.SchemaDefinitionError,
+        "module 'super' is a reserved module name"
+    )
+    def test_schema_module_reserved_01(self):
+        """
+            module foo {
+                module super {}
+            }
+        """
+
+    @tb.must_fail(
+        errors.SchemaDefinitionError,
+        "module 'ext' is a reserved module name"
+    )
+    def test_schema_module_reserved_02(self):
+        """
+            module foo {
+                module ext {}
+            }
+        """
+
```

Comment:
We have a list of reserved module names with explanations of why they are reserved (see bottom of https://www.edgedb.com/docs/datamodel/index#terminology). These two are not currently on that list, so I'd like to add them. What are the reasons for `super` and `ext` being reserved? What do we use those for?

I'm guessing `super` lets us reference something in the parent, and I believe I recall mention of `ext` being for Postgres extensions? I may be confusing that with something else.

Reply:
`super` is reserved for future use, for reference to the parent module:

```
module hello {
   type Hello;

   module world {
       type World {
           link hello -> super::Hello;
       }
   }
}
```
(I don't think this is implemented yet, but it's a reasonable design)

`ext` is for extensions in general. And extensions in EdgeDB are not the same thing as Postgres extensions (as I initially believed).